styleguide/source/_patterns/03-organisms/footer.twig

```diff
@@ -14,7 +14,7 @@
             <ul>
               {% for link in column %}
                 <li>
-                  <a href="{{ link.url }}" class="ama__link--black">{{ link.text }}</a>
+                  {% include "@atoms/link.twig" %}
```

Comment:
Go ahead and set the class here so that its `{% include "@atoms/link.twig" with "class" : "ama__link--black" %}`